### ISSUE TITLE
Issue with 'training' paramter for batchnormalization of 3d convolution

### ISSUE THREAD

**Issue by shreyas1998** [qianguih/voxelnet#84]
The  batchnormalization of 'conv1' , 'conv2' and 'conv3' in model/rpn.py are set to training phase even during testing?
Shouldn't training=self.training be given as an additional argument?
![image](https://user-images.githubusercontent.com/34686465/84898708-50458280-b0c5-11ea-8a3b-c4776cd3e6c7.png)
